src/components/ranking-chart/ranking-chart.js

```diff
@@ -14,35 +11,27 @@ import { openInfoModalAnalyticsEvent } from 'actions/google-analytics-actions';
 const actions = {...metadataActions, ...urlActions, openInfoModalAnalyticsEvent };
 
 const RankingChartContainer = (props) => {
-  const handleInfoClick = () => {
-    const { setModalMetadata, openInfoModalAnalyticsEvent } = props;
-    openInfoModalAnalyticsEvent('Countries ranking');
-    const md = metadataConfig[RANKING_CHART];
-    setModalMetadata({
-      slug: md.slug,
-      title: md.title,
-      isOpen: true
-    });
-  }
-
-  const handleSortClick = (category) => {
-    const { changeUI, sortRankingCategory } = props;
-    const sortedCategory = sortRankingCategory && sortRankingCategory.split('-')[0];
-    const direction = sortRankingCategory && sortRankingCategory.split('-')[1];
-    let sortDirection = sortedCategory === category && direction === SORT.ASC ? SORT.DESC : SORT.ASC;
-    changeUI({ sortRankingCategory: `${category}-${sortDirection}` })
+  const handleFilterSelection = (selectedFilter) => {
+    const { changeUI } = props;
+    changeUI({ sortRankingCategory: selectedFilter })
   }
 
   const handleCountryClick = (countryISO, countryName) => {
     const { changeGlobe } = props;
     changeGlobe({ countryISO, countryName, zoom: null, center: null });
   };
 
+  const handleSearchChange = (event) => {
```

Comment:
I'd say that we should `debounce` firing the search change function, to avoid triggering it on each key stroke